### PR TITLE
Am form

### PR DESCRIPTION
**Pull request by aemcdonald**
### Code Highlights:
 * This PR addresses functionality for the question form. The form will progress through questions on next click & will store the selected date in state. 
 
### Where should the reviewer start?
 * FormContainer.tsx line 1
 * Question.tsx line 1
 
### Have Tests Been Added?
- [X] No.
- [ ] Yes, but all tests are not passing.
- [ ] Yes, and all are passing.

### Any background context you want to provide?
Currently there is no sad path handling & sample questions are hard-coded in. Need to clear form after submit & add functionality to skip button. 

### Message/Questions for reviewer:
N/A

### Issues:
* Addresses #25

### Screenshots (if appropriate):
N/A

### Tracking Consistency:
- [X] added appropriate labels
- [X] My code follows the code style of this project and has removed all unnecessary annotations
- [ ] I have added comments on my pull request, particularly in hard-to-understand areas
- [X] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

**Review by krogowsk531 — APPROVED** [review]
Great job on working with TypeScript on this first component!

File: src/Components/FormContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useState } from 'react';
+import { Question } from './Question';
+
+export const FormContainer = () => {
+  let questions = ['who', 'what', 'where', 'when', 'why'];
+  //const [questions, setQuestions] = useState([])
+
+  useEffect(() => {
+    //set questions & store in questions variable
```
Comment (taylorjohnson141) [inline src/Components/FormContainer.tsx:9]: This is where we will fetch the questions from the backend!

File: src/Components/FormContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  const [answers, setAnswer] = useState<string[]>([]);
+
+  const [currentQuestionIndex, changeQuestion] = useState(0)
+
+  return (
+    <>
+      <h1>Create My Calendar</h1>
+      <form>
+        <Question currentQuestion={questions[currentQuestionIndex]} changeQuestion={changeQuestion} currentQuestionIndex={currentQuestionIndex} setAnswer={setAnswer} answers={answers}/>
```
Comment (taylorjohnson141) [inline src/Components/FormContainer.tsx:21]: Is there any way we can avoid using so many props?

File: src/Components/Question.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  currentQuestionIndex: number;
+  currentQuestion: string;
+  setAnswer: SetAnswer;
+  answers: string[];
+};
+
+export const Question = (props: QuestionProps) => {
+  const [answer, saveAnswer] = useState('');
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```
Comment (taylorjohnson141) [inline src/Components/Question.tsx:18]: We need to see if this actually updates the answers correctly!

**Review by taylorjohnson141 — APPROVED** [review]
Looks really good thank you for starting

**Pull request merged**